transactional.md

```diff
@@ -0,0 +1,79 @@
+# Challenges Administering Transactional Systems
+
+## Software Installation
+
+There are situations in which Cockpit or the containerized version of YaST would need to install
+software in the system being managed.
```

Comment:
maybe I would write `in the managed system`. this one sound a bit overcomplicated for me